.github/workflows/publish-prisma-fmt-wasm.yml

```diff
@@ -0,0 +1,68 @@
+name: Build and publish @prisma/prisma-fmt-wasm
+
+concurrency: build-prisma-fmt-wasm
+
+on:
+  workflow_dispatch:
+    inputs:
+      enginesWrapperVersion:
+        required: true
+      enginesHash:
+        required: true
+      npmDistTag:
+        required: true
+        default: 'latest'
+
+jobs:
+  build:
+    name: Build and publish @prisma/prisma-fmt-wasm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Print input
+        env:
+          THE_INPUT: '${{ toJson(github.event.inputs) }}'
+        run: |
+          echo $THE_INPUT
+
+      - uses: actions/checkout@v3
+        ref: ${{ github.event.inputs.enginesHash }}
+      - uses: cachix/install-nix-action@v16
+
+      #
+      # Build
+      #
+
+      - run: nix build .#prisma-fmt-wasm
```

Comment:
Not necessary, but if there is a build problem, it's cleaner if it happens in its own step. The build is cached for the next step.